contracts/utils/convex/ConvexStakingWrapperYield.sol

```diff
@@ -0,0 +1,111 @@
+//https://etherscan.io/address/0x3ba207c25a278524e1cc7faaea950753049072a4#code
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+pragma experimental ABIEncoderV2;
+
+import './ConvexStakingWrapper.sol';
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+
+struct Balances {
+    uint128 art; // Debt amount
+    uint128 ink; // Collateral amount
+}
+
+interface ICauldron {
+    /// @dev Each vault records debt and collateral balances_.
+    function balances(bytes12 vault) external view returns (Balances memory);
+}
+
+//Staking wrapper for Yield platform
+//use convex LP positions as collateral while still receiving rewards
+contract ConvexStakingWrapperYield is ConvexStakingWrapper,AccessControl {
```

Comment:
You don't actually need `AccessControl`. The parent class uses `Ownable`, which we could reuse if we would need to. If you keep reading you'll see we need no permissioned functions.